packages/testlab/src/test-sandbox.ts

```diff
@@ -87,4 +94,36 @@ export class TestSandbox {
       : resolve(this.path, parse(src).base);
     await copy(src, dest);
   }
+
+  /**
+   * Copies a file from src to the TestSandbox. `.js.map` files are also copied
+   * to the sandbox by default.
+   * @param src Absolute path of file to be copied to the TestSandbox
+   * @param [dest] Optional. Destination path / filename of the copy operation
+   * (relative to TestSandbox). Original filename used if not specified or path
+   * given.
+   * @param copyMap Copy `.js.map` to TestSandbox or not. Defaults to true.
+   */
+  async copySourceCodeFile(
+    src: string,
+    dest?: string,
+    copyMap: boolean = true,
```

Comment:
In general: Boolean arguments are evil, please use an options object `{copyMap: boolean}` instead.

In this particular use case, I think `copyMap` option is [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). Since your code is already handling the case where no `.js.map` file is available (and skip copying), I think this is all we need for now.